examples/cvae_dim_example/client.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+from pathlib import Path
+from typing import Sequence, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+import torchvision.transforms as transforms
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from examples.cvae_dim_example.mnist_model import MnistNet
+from fl4health.clients.basic_client import BasicClient
+from fl4health.pipeline.autoencoder_pipeline import CVAEPipeline
+from fl4health.utils.load_data import load_mnist_data
+from fl4health.utils.metrics import Accuracy, Metric
+from fl4health.utils.random import set_all_random_seeds
+from fl4health.utils.sampler import DirichletLabelBasedSampler
+
+
+class CVAEDimClient(BasicClient):
+    def __init__(self, data_path: Path, metrics: Sequence[Metric], DEVICE: torch.device, condition: str):
+        BasicClient.__init__(self, data_path, metrics, DEVICE)
+        self.processing = CVAEPipeline(condition)
```

Comment:
In above example this was named as self.training_pipeline, is there a reason that you named it self.processing here?
Isn't it better to keep them same for consistency?

Reply:
We can potentially use a name that makes sense in both examples like 'cvae_pipeline' or 'pipeline', however, I wanted to highlight the differences between the usage of this object in the two examples as the usage is quite different.
In this example, the object from the CVAEPipeline class is only used for dimensionality reduction, so it is a pre-processing kind of object. while, in the other examples, an object from the same class is used only for training the (C)VAE. In this example we are not doing any CVAE training, therefore, I think the name `training_pipeline` is not very suitable. The same applies to the other examples. Please let me know if you think it is still better to have consistency.